src/classes/ADDR_Contact_TDTM.cls

```diff
@@ -49,507 +49,450 @@ public class ADDR_Contact_TDTM extends TDTM_Runnable {
     * @param objResult the describe for Contacts 
     * @return dmlWrapper.  
     ********************************************************************************************************/
-    public override DmlWrapper run(List<SObject> listNew, List<SObject> listOld, 
-        TDTM_Runnable.Action triggerAction, Schema.DescribeSObjectResult objResult) {
+    DmlWrapper dmlWrapper = new DmlWrapper();
```

Comment:
This class is still currently in progress.  Continuing to split and pull the logic out, pushing it mostly into the Contacts class.  More changes on the way.